odoo/addons/base/models/ir_ui_view.py

```diff
@@ -398,9 +398,9 @@ def _check_xml(self):
                     check = valid_view(view_arch, env=self.env, model=view.model)
                     view_name = ('%s (%s)' % (view.name, view.xml_id)) if view.xml_id else view.name
                     if not check:
-                        raise ValidationError(_('Invalid view %s definition in %s') % (view_name, view.arch_fs))
+                        raise ValidationError(_('Invalid view %s definition in %s', view_name, view.arch_fs))
```

Comment:
named placeholders could be nice here

Reply:
I think this could be a general advice. Since this is fixing translation, why not give a good example and use named parameter everywhere you make some change if possible :)